### PR TITLE
Knockout cleanup

### PR DESCRIPTION
**Pull request by orangejenny**
Commits should be self-explanatory.

@czue / @esoergel

### CODE REVIEW

File: corehq/apps/style/static/style/ko/knockout_bindings.ko.js [hunk 1]
```diff
@@ -1,5 +1,39 @@
 /* global DOMPurify */
 
+ko.bindingHandlers.hqbSubmitReady = {
+    update: function(element, valueAccessor) {
+        var value = (valueAccessor()) ? valueAccessor()() : null;
+        if (value)
+            $(element).addClass("btn-primary").removeClass("disabled");
+        else
+            $(element).addClass("disabled").removeClass("btn-primary");
+    }
```
Comment (dimagimon) [inline corehq/apps/style/static/style/ko/knockout_bindings.ko.js]: Missing trailing comma. (comma-dangle)

File: corehq/apps/style/static/style/ko/knockout_bindings.ko.js [hunk 2]
```diff
@@ -1,5 +1,39 @@
...
+ko.bindingHandlers.fadeVisible = {
+    // from knockout.js examples
+    init: function(element, valueAccessor) {
+        var value = valueAccessor();
+        $(element).toggle(ko.utils.unwrapObservable(value));
+    },
+    update: function(element, valueAccessor) {
+        var value = valueAccessor();
+        ko.utils.unwrapObservable(value) ? $(element).fadeIn() : $(element).fadeOut();
+    }
```
Comment (dimagimon) [inline corehq/apps/style/static/style/ko/knockout_bindings.ko.js]: Missing trailing comma. (comma-dangle)

File: corehq/apps/style/static/style/ko/knockout_bindings.ko.js [hunk 3]
```diff
@@ -1,5 +1,39 @@
...
+ko.bindingHandlers.fadeVisibleInOnly = {
+    // from knockout.js examples
+    init: function(element, valueAccessor) {
+        var value = valueAccessor();
+        $(element).toggle(ko.utils.unwrapObservable(value));
+    },
+    update: function(element, valueAccessor) {
+        var value = valueAccessor();
+        ko.utils.unwrapObservable(value) ? $(element).fadeIn() : $(element).hide();
+    }
```
Comment (dimagimon) [inline corehq/apps/style/static/style/ko/knockout_bindings.ko.js]: Missing trailing comma. (comma-dangle)

**Review by benrudolph — APPROVED** [review]
nice

**Pull request merged**